readme.md

```diff
@@ -22,13 +22,19 @@ gravatarUrl('sindresorhus@gmail.com', {size: 200});
 
 ## API
 
-### gravatarUrl(email, [options])
+### gravatarUrl(identifier, [options])
 
-#### email
+#### identifier
 
 Type: `string`
 
-Email matching a Gravatar profile.
+Identifier for which to get the Gravatar picture.
+
+This will typically be an email matching a Gravatar profile, but can technically be any string.
+
+The Gravatar service only sees a hash of the identifier, so you could actually use this to get pseudo-random avatars for any entity, e.g. based on its ID.
+
+Note that if the identifier contains an `@`, it is assumed to be an email, and will therefore be lower-cased and trimmed before hashing, as per the Gravatar instructions - otherwise it will be hashed as-is.
```

Comment:
Can you add this text to the index.d.ts file? I prefer them to be in sync to ease updates.

Reply:
Done :-)